default/AI/FreeOrionAI.py

```diff
@@ -166,6 +166,7 @@ def generateOrders():  # pylint: disable=invalid-name
     random_seed = str(fo.getGalaxySetupData().seed) + "%03d%05d" % (fo.empireID(), turn)
     random.seed(random_seed)
     aggression_name = fo.aggression.values[foAIstate.aggression].name
+
```

Comment:
yes, that's a comment I already made on the other PR, but I didn't want to ask luiges to do further cleanup of history before getting things going here.